src/test/regress/citus_tests/config.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, arguments):
             "max_connections": 1200,
         }
         self.new_settings = {}
-        self.add_coordinator_to_metadata = False
+        self.add_coordinator_to_metadata = True
```

Comment:
we can get rid of the property self.`add_coordinator_to_metadata` from configs as they would be always True from now on.